README.md

```diff
@@ -1,32 +1,36 @@
-# Docksal [![Latest Release](https://img.shields.io/github/release/docksal/docksal.svg?style=flat-square)](https://github.com/docksal/docksal/releases/latest) [![Setup Instructions](https://img.shields.io/badge/%E2%9A%99-%20Setup%20Instructions%20-blue.svg)](http://docksal.readthedocs.io/en/master/getting-started/env-setup)
+# Docksal [![Latest Release](https://img.shields.io/github/release/docksal/docksal.svg?style=flat-square)](https://github.com/docksal/docksal/releases/latest) [![Setup Instructions](https://img.shields.io/badge/%E2%9A%99-%20Setup%20Instructions%20-blue.svg)](http://docksal.readthedocs.io/en/master/getting-started/setup)
 
 [![Build Status](https://img.shields.io/travis/docksal/docksal.svg?style=flat-square)](https://travis-ci.org/docksal/docksal)
 [![Docs](https://readthedocs.org/projects/docksal/badge?version=master&style=flat-square)](http://docksal.readthedocs.io/en/master)
 [![Gitter](https://img.shields.io/gitter/room/docksal/community-support.svg?style=flat-square)](https://gitter.im/docksal/community-support)
 
 Docker powered environments for web development on macOS, Windows, and Linux.
 
-Running a complete LAMP stack for your project can be two steps away!<sup>*</sup>
+Running a complete LAMP stack for your project can be one step away!<sup>*</sup>
 
 ```
-git clone <sample-project-repo>
-fin init
+fin project create
 ```
 
-<sup>*</sup>Once you are done with a one-time [Docksal environment setup](http://docksal.readthedocs.io/en/master/getting-started/env-setup)
+<sup>*</sup>Once you are done with a one-time [Docksal setup](http://docksal.readthedocs.io/en/master/getting-started/setup)
```

Comment:
Should these links to other doc pages be relative rather than hard-coded? Users can be on docs.docksal.io and then moved to readthedocs or vice-versa.

Reply:
This file is only visible on GitHub

Reply:
But is makes sense to update those links to point to docs.docksal.io.